src/test/data/script_valid.json

```diff
@@ -690,16 +690,6 @@
     "2-of-2 CHECKMULTISIG NOT with both pubkeys valid, but second signature invalid. Valid pubkey fails, and CHECKMULTISIG exits early, prior to evaluation of second invalid signature."
 ],
 
```

Comment:
Many previously valid test cases were testing the now-unsupported no-DERSIG option. These have to be removed. This isn't reducing useful test coverage because that option was never used for the Zcash block chain.